exotations/solvers/exotica_levenberg_marquardt_solver/CMakeLists.txt

```diff
@@ -23,3 +25,7 @@ install(TARGETS ${PROJECT_NAME}
   LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
   RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
 install(FILES exotica_plugins.xml DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+
+else()
+message("The Levenberg-Marquardt solver requires at least Eigen version 3.3.")
```

Comment:
Please make it a `WARNING` - otherwise it gets hidden by `catkin build -s`.

Reply:
Also, change to `if(NOT TARGET ...) message(WARNING...) return() endif()` - no else block required and can be removed later on much more easily.

Reply:
Nice, I didn't know that you can directly return from a CMake script file.